app/views/shared/fields/_title.html.erb

```diff
@@ -34,11 +34,11 @@
         subtitleVal += (subtitleVal.blank? ? "" : ", ") + translationVariant
       end
       
-      if !result['value'].blank?
-         title = getFullTitle result
+      if !result['name'].blank?

```

Comment:
@mcritchlow Yes the full suite passed for me when i run from my local. It seems like that Arwen agreed to use the value in 'name' (authoritativeLabel) as the first place.  Should I ping her again?